### PR TITLE
311 issue error nckliprotobufloadgenerator please make sure that properties data type and expression function return type are compatible with each other

### PR DESCRIPTION
**Pull request by mseijasm**

### CODE REVIEW

**Comment by RaulBejaranoSngular** [comment]
Codacy static code analisis is complaining about line sizes and duplication of code. Please, check this before merge. [Click here for more details](https://app.codacy.com/gh/corunet/kloadgen/pullRequest?prid=10677854&bid=32446707)

**Review by RaulBejaranoSngular — DISMISSED** [review]
As we talk earlier this is the easyest solution without making big changes. It`s ok to me but wait for @kszosze approval to merge.

**Review by dianaGomezGar — DISMISSED** [review]

**Review by pedror19 — APPROVED** [review]

**Review by davidgarciago — APPROVED** [review]

**Review by pedror19 — APPROVED** [review]

**Review by dianaGomezGar — APPROVED** [review]

**Review by davidgarciago — APPROVED** [review]

**Review by dianaGomezGar — APPROVED** [review]

**Review by pedror19 — APPROVED** [review]

**Review by dianaGomezGar — APPROVED** [review]

**Review by pedror19 — DISMISSED** [review]

**Review by jemacineiras — APPROVED** [review]

**Review by pedror19 — DISMISSED** [review]

**Review by jemacineiras — APPROVED** [review]

**Pull request merged**
